src/app/functions/server/onerep.ts

```diff
@@ -398,24 +396,46 @@ export async function getScanDetails(
 export async function getAllScanResults(
   profileId: number,
 ): Promise<ScanResult[]> {
-  const scanPagesAll = [];
-  const firstPage = await listScanResults(profileId, {
-    per_page: 100,
+  return fetchAllPages((page: number) =>
+    listScanResults(profileId, { per_page: 100, page: page }),
+  );
+}
+
+export async function getAllDataBrokers() {
+  return fetchAllPages(async (page: number) => {
+    const response = await onerepFetch(
+      "/data-brokers?per_page=100&page=" + page.toString(),
```

Comment:
Yeah, I generally agree in cases where we accept arbitrary URL params and want to make sure they're formatted correctly, but in this case, I was afraid that it would only add noise for what is essentially still a fairly simple URL.